src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/Behavior/DesignerActionGlyph.cs

```diff
@@ -98,12 +102,12 @@ public bool IsInComponentTray
     private Image GlyphImageClosed => _glyphImageClosed ??= ScaleHelper.GetIconResourceAsBitmap(
         typeof(DesignerActionGlyph),
         "Close_left",
-        ScaleHelper.InitialSystemDpi);
+        new Size(_iconsWidth, _iconsHeight));
```

Comment:
ScaleHelper.ScaleToInitialSystemDpi(IconSize)
```suggestion
        ScaleToDpi(new Size(IconSize, IconSize),  ScaleHelper.InitialSystemDpi));
```